hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/utils/UriUtils.java

```diff
@@ -73,6 +98,63 @@ public static String generateUniqueTestPath() {
     return testUniqueForkId == null ? "/test" : "/" + testUniqueForkId + "/test";
   }
 
+  public static String maskUrlQueryParameters(List<NameValuePair> keyValueList,
+      HashSet<String> queryParamsForFullMask,
```

Comment:
and have these methods take Set<> too. Just to hide the detail that HashSet is used

Reply:
done